schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -62,6 +62,46 @@
                   { "$ref": "http://schema.management.azure.com/schemas/2014-06-01/Microsoft.Web.json#/resourceDefinitions/serverfarms" },
                   { "$ref": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Web.json#/resourceDefinitions/certificates" },
                   { "$ref": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Web.json#/resourceDefinitions/serverfarms" },
```

Comment:
e.g. In schemas/2015-01-01/deploymentTemplate.json, we have this entry "2015-08-01/Microsoft.Web.json#/resourceDefinitions/serverfarms". But in schemas/2014-04-01-preview, it's not there.